### PR TITLE
RPC call for adding UTXOTags

### PR DESCRIPTION
**Pull request by markettes**

### CODE REVIEW

File: src/Rpc/NodeGuardService.cs [hunk 1]
```diff
@@ -994,7 +996,45 @@ public override async Task<GetChannelResponse> GetChannel(GetChannelRequest requ
...
+            if (utxo.Length != 2)
+            {
+                throw new RpcException(new Status(StatusCode.InvalidArgument, $"Invalid outpoint {tag.UtxoOutpoint}"));
+            }
+
+            if (!uint.TryParse(utxo[1], out _))
+            {
+                throw new RpcException(new Status(StatusCode.InvalidArgument, $"Invalid output index {tag.UtxoOutpoint}"));
+            }
+        }
```
Comment (RodriFS) [inline src/Rpc/NodeGuardService.cs:1019]: you can validate the outpoint directly with `OutPoint.TryParse()`

**Review by RodriFS — DISMISSED** [review]

File: src/Proto/nodeguard.proto [hunk 2]
```diff
@@ -68,6 +68,11 @@ service NodeGuardService {
   Gets a channel by id
    */
   rpc GetChannel(GetChannelRequest) returns (GetChannelResponse);
+  
+  /*
+  Adds tags
```
Comment (Jossec101) [inline src/Proto/nodeguard.proto:73]:
```suggestion
  Adds tags to UTXOs in the system as
```

File: src/Rpc/NodeGuardService.cs [hunk 3]
```diff
@@ -994,7 +996,45 @@ public override async Task<GetChannelResponse> GetChannel(GetChannelRequest requ
 
         return result;
     }
-    
+
+    public async Task<AddTagsResponse> AddTags(AddTagsRequest request, ServerCallContext context)
```
Comment (Jossec101) [inline src/Rpc/NodeGuardService.cs:1000]:
```suggestion
    public override async Task<AddTagsResponse> AddTags(AddTagsRequest request, ServerCallContext context)
```

File: src/Rpc/NodeGuardService.cs (hunk 1 above)
Comment (Jossec101) [reply src/Rpc/NodeGuardService.cs:1019]: @RodriFS can you implement that?

**Review by Jossec101 — APPROVED** [review]

**Pull request merged**